app/controllers/application_controller/timelines.rb

```diff
@@ -128,41 +128,7 @@ def tl_chooser
   end
 
   def getTLdata
```

Comment:
I have no idea why it's named so...interestingly.  I'm fine with renaming, I'll take care of that now.